src/react/components/pages/export/exportForm.tsx

```diff
@@ -160,8 +159,9 @@ export default class ExportForm extends React.Component<IExportFormProps, IExpor
 
         const formData = { ...exportFormat };
         if (resetProviderOptions) {
-            formData.providerOptions = { assetState: ExportAssetState.All };
+            formData.providerOptions = { assetState: ExportAssetState.Visited };
```

Comment:
> @mydiemho What do you mean? This should make all the difference regarding how the form controls get initialized.

@wbreza  what I mean is that this is the code that set the `Provider Options` to the right default when I switch `Provider Type`.  The `default` in json file doesn't do anything